events.go

```diff
@@ -196,6 +200,10 @@ func (s *eventStatusReport) Duration() time.Duration {
 	return s.finishTime.Sub(s.startTime)
 }
 
+func (s *eventStatusReport) FlushDuration() time.Duration {
```

Comment:
Could you add a comment that mentions the flush duration is time from flush start to rotating out the buffers, not necessarily just the latency of the network call to flush spans?

Reply:
sg, though i think it won't be too helpful to end users unless we add a comment to the interface too - wdyt?

Reply:
@bg451, I added godoc comments for each field on the interface, wdyt?

Reply:
yep! This is great, thanks!